README.md

```diff
@@ -4,7 +4,34 @@ In this repository, we collect example scripts that can be useful for the team a
 
 ## Prerequisites
 
-TBD
+To run python scripts inside `examples` directory you should first run the installer script:
+
+```bash
+$ bin/install
+```
+
+This will handle installing `venv` and requierd packages.
+
+After the installtion, you can run the scripts using `bin/run` script:
+
+```bash
+$ bin/run <path_to_script_file> <arguments>
+```
+
+### Authorization
+
+In order to authorize the requests, please add your credentails in a `JSON` format into the `.config` file:
```

Comment:
Shouldn't we mention https://community.canvaslms.com/t5/The-Product-Blog/Connecting-Studio-OAuth-via-Postman/ba-p/259739 ?

Reply:
We can! I didn't know about this link, added